Minor Drink Tweaks

Sweedle opened the pull request:
🆑 Sweedle
bugfix: Added a recipe for Kamikaze! Combine 1 part vodka, triple sec and lime juice for a sour treat.
tweak: Cherry and strawberry soda are now made with cherry jelly and berry juice respectively.
bugfix: Hooch can now be made as normal. Fringe weaver is now made with 2 parts ethanol, 1 part sugar and 1 part water.
tweak: Fringe weaver will now get you significantly drunker.
/🆑

~~Technically conflicts with #30527 - @jwc1015 if you want me to add your tweak to this so you don't have to squash, lmk.~~

Kamikaze was just sort of... hanging out in the code without a recipe, so I gave it one.

Fixes #30526

MuckerMayhem commented on code/modules/reagents/Chemistry-Recipes.dm:
```diff
@@ -2622,6 +2622,12 @@
+/datum/chemical_reaction/kamikaze
+	name = "Kamikaze"
+	result = /datum/reagent/ethanol/kamikaze
+	required_reagents = list(/datum/reagent/ethanol/vodka = 1,/datum/reagent/ethanol/triple_sec = 1, /datum/reagent/drink/juice/lime = 1)
```
```suggestion
	required_reagents = list(/datum/reagent/ethanol/vodka = 1, /datum/reagent/ethanol/triple_sec = 1, /datum/reagent/drink/juice/lime = 1)
```
A missing space? unbelievable...

Sweedle replied:
me when a missing space: 😔😔😔😔

Sweedle committed:
Minor Drink Tweaks

- Add Kamikaze
- Fix cherry/berry soda mixup
- Fix fringe weaver strength
Minor drink tweaks 2

Fringe weaver, kamikaze
MDT3

Fringe weaver output

SierraKomodo commented:
Wiki updated.
